napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -393,19 +429,6 @@ def _set_keybinding(self, row, col):
         self._table.setCurrentItem(self._table.item(row, col))
 
         if col in {self._shortcut_col, self._shortcut_col2}:
-            # Get all layer actions and viewer actions in order to determine
-            # the new shortcut is not already set to an action.
```

Comment:
Sorry, probs a silly question but why is it ok to now remove this code?

Reply:
Oh wait lmao 🤦‍♀️, it's cause of [this comment](https://github.com/napari/napari/pull/7231#issuecomment-2326959136) above right?